frontend/src/components/organisms/EventForm.tsx

```diff
@@ -169,12 +181,22 @@ const EventForm = ({
           startTime,
           endTime,
         } = data;
+        const userid = await fetchUserIdFromDatabase(user?.email as string);
         const startDateTime = convertToISO(startTime, startDate);
         const endDateTime = convertToISO(endTime, startDate);
+        let imageURL = null;
+        if (typeof eventId === "string") {
+          imageURL = await fetchImageURLFromDB(eventId); // Assign imageURL to previous URL.
+        }
```

Comment:
Why do we need a separate query to get the imageUrl from Db if it is already present in the event object?

Reply:
Because I could not find a way to update imageURL accordingly to the form in event form. I tried set state, but it does not work properly. And I could not figure out how I could integrate it with dropzone. So I do this workaround instead.